### PR TITLE
Allows readying up in lobby, the game doesn't start as soon as it can anymore

### PR DESCRIPTION
**Pull request by ivanmixo**
## About the Pull Request

- Title

## Why It's Good For The Game

Lobby gud

## Changelog

:cl:
qol: You can now actually ready up before the game starts
/:cl:

### CODE REVIEW

File: code/controllers/subsystems/ticker.dm [hunk 1]
```diff
@@ -39,7 +39,8 @@ SUBSYSTEM_DEF(ticker)
 	var/round_start_time = 0
 
 /datum/controller/subsystem/ticker/Initialize()
-	to_world("<span class='info'><B>The game will start soon!</B></span>")
+	to_world("<span class='info'><B>Welcome to the pre-game lobby!</B></span>")
+	to_world("Please, setup your character and select ready. Game will start in [round(pregame_timeleft/10)] seconds.")
```
Comment (Tsurupeta) [inline code/controllers/subsystems/ticker.dm]:
```suggestion
	to_world("Please, setup your character and select ready. Game will start in [round(pregame_timeleft/10)] seconds.")
```

maybe we could do a little `if(!start_ASAP) to_world...

Comment (ivanmixo) [reply code/controllers/subsystems/ticker.dm]: Bleh fine

File: code/modules/mob/new_player/menu.dm [hunk 2]
```diff
@@ -85,23 +85,42 @@
...
+	if(GAME_STATE <= RUNLEVEL_LOBBY)
+		player.ready = !player.ready
+		to_chat(player, "<span class='notice'>You are now [player.ready ? "ready" : "not ready"].</span>")
+
+	else
+		player.LateChoices() //show the latejoin job selection menu
+
+	update_icon(player)
+
+/obj/screen/new_player/selection/join_game/update_icon(mob/new_player/player)
```
Comment (Vivalas) [inline code/modules/mob/new_player/menu.dm:115]: this doesn't actually work, I tested locally

**Review by Vivalas — CHANGES_REQUESTED** [review]

File: code/modules/mob/new_player/menu.dm (hunk 2 above)
Comment (Vivalas) [reply code/modules/mob/new_player/menu.dm:115]: as in you have to relog for it to take effect

Comment (ivanmixo) [reply code/modules/mob/new_player/menu.dm:115]: Works fine on my end, minus it not automatically updating when round starts

**Comment by ivanmixo** [comment]
Waiting for the DCS PR.

**Pull request closed without merging**